src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx

```diff
@@ -135,7 +145,11 @@ export class ArtworkSidebarCurrentBidInfo extends React.Component<
               <Serif size="2" color="black60">
                 <br />
                 This auction has a{" "}
-                <Link onClick={this.onClickBuyerPremium.bind(this, mediator)}>
+                <Link
+                  onClick={() => {
+                    this.onClickBuyerPremium(mediator)
```

Comment:
Since `onClickBuyerPremium` doesn't actually need any e.MouseEvent props that would be passed along by doing
```js
onClick={this.onClickBuyersPremium}
```
this is totally fine.